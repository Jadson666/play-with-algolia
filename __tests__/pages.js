import React from 'react'
import renderer from 'react-test-renderer'
import SearchPage from '../src/pages/search'
import FavoritePage from '../src/pages/favorite'

it('test search page is unchanged', () => {
  const tree = renderer.create(<SearchPage />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('test favorite search page is unchanged', () => {
  const tree = renderer.create(<FavoritePage />).toJSON()
  expect(tree).toMatchSnapshot()
})
