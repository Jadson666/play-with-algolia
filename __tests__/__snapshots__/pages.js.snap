// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test favorite search page is unchanged 1`] = `
<div>
  <div
    className="important"
  >
    Try to add your Favorite :)
  </div>
</div>
`;

exports[`test search page is unchanged 1`] = `
<div
  style={
    Object {
      "padding": "0 30px",
    }
  }
>
  <div
    className="inputContainer"
  >
    <input
      className="searchInput"
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div>
    <div
      className="important"
    >
      No Result
    </div>
  </div>
</div>
`;
